test/snippet/alphabet/aminoacid/enable_aminoacid.cpp

```diff
@@ -0,0 +1,34 @@
+#include <seqan3/alphabet/aminoacid/concept.hpp>
+
+namespace your_namespace
+{
+
+// your own aminoacid definition
+struct your_aa : seqan3::aminoacid_empty_base
+{
+    //...
+};
+
+}
```

Comment:
```suggestion
} // namespace your_namespace
```

Reply:
I had it like that but it looked very noisy, is it really helpful for such a small example?

Reply:
I just marked it because we have these comments everywhere and it turned kind of into a code style. But for these short snippets I agree that we don't need them.